tests/test_mock.py

```diff
@@ -59,9 +57,8 @@ class MockRunWithStop(RunActor):
     def __init__(self, callback):
         db = MockDB()
         super(MockRunWithStop, self).__init__(db)
-
         self.callback = callback
+        self.iterator = EndlessViewIterator(self.iterator)
```

Comment:
Is this an OK assumption? Its similar to `stop_callback` functionality change in the `RunActor` class: users can also use the TaskView or PrioritizedView.